app/javascript/article-form/articleForm.jsx

```diff
@@ -1,8 +1,11 @@
 import 'preact/devtools';
 import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
 import linkState from 'linkstate';
 import postscribe from 'postscribe';
+// eslint-disable-next-line import/no-unresolved
 import ImageUploadIcon from 'images/image-upload.svg';
+// eslint-disable-next-line import/no-unresolved
 import ThreeDotsIcon from 'images/three-dots.svg';
```

Comment:
It says, it cannot resolve the path in `images/...`. This folder doesn't exist indeed. I'm guessing it's gonna be created during a building process. Can anyone explain it to me?

Reply:
Hi, in https://github.com/thepracticaldev/dev.to/blob/master/config/webpacker.yml#L11 the folder `app/assets` is added to the resolve path, `images/three-dots.svg` and the others are located in that folder.

Reply:
I see, but Eslint is not aware of that and complaining about it. Is it ok to use the "disable next line" back door?

Reply:
Sure, in this case makes total sense